stylesheets/_component.rules.scss

```diff
@@ -493,6 +535,7 @@ td.table__form .rules + .form__group:last-of-type {
     }
 
     .rule-block__heading {
+        font-weight: bold;
```

Comment:
preferred over `$font-family-regular` ?

Reply:
Used to avoid contrast warnings in SI. Alternative would be to change logic bg colours.